bin/v-add-web-domain

```diff
@@ -94,10 +94,8 @@ check_hestia_demo_mode
 # Reading user values
 source_conf "$USER_DATA/user.conf"
 
-[[ -e "$HOMEDIR/$user/web/$domain" ]] && check_result "$E_EXISTS" "Web domain folder for $domain should not exist"
```

Comment:
With out the check it is prossible to create a symbolic link by the user and chown on line 101 give will give the permissions of all files in that folder...

Reply:
Okay, I accept. I changed the condition to a link check. :-)

Reply:
Users can still create the folder and do the same trick with symbolic link 

We probally need to change the check to:

[[ -L "$HOMEDIR/$user/web/$domain" || -d "$HOMEDIR/$user/web/$domain"  ]]  && check_result "$E_EXISTS" "Web domain folder for $domain should not exist"

then

Reply:
After this conversation, I think the best is not merging this pull request. It is a more complicated condition... :-)